scripts/regression/test_runner.py

```diff
@@ -175,4 +191,20 @@ class BenchmarkResult:
 # nuke cached benchmark data between runs
 if os.path.isdir("duckdb_benchmark_data"):
     shutil.rmtree('duckdb_benchmark_data')
+
+if summary and not no_summary:
+    print(
+        '''\n\n====================================================
+================  FAILURES SUMMARY  ================
+====================================================
+'''
+    )
+    for i, failure_message in enumerate(summary, start=1):
+        print(f"{i}: ", failure_message["benchmark"])
+        if failure_message["old_failure"] != failure_message["new_failure"]:
+            print("Old:\n", failure_message["old_failure"])
+            print("New:\n", failure_message["new_failure"])
+        else:
+            print(failure_message["old_failure"])
+
```

Comment:
Here format is different that above, with 3 lines instead of single line, maybe the can be made somewhat more similar?

Also here it's `f{i}: ` above `f{i}. `, I would try to set up on a standard to follow.